structs/imputation/ImputationBeagleStructs.wdl

```diff
@@ -0,0 +1,8 @@
+version 1.0
+
+struct ReferencePanelContig {
```

Comment:
ah is this how its done in warp? neat

Reply:
yeah, not sure why, but happy to copy them